pkg/controller/nodefeaturediscovery/nodefeaturediscovery_controls.go

```diff
@@ -34,6 +34,42 @@ func (s ResourceStatus) String() string {
 	return names[s]
 }
 
+func Namespace(n NFD) (ResourceStatus, error) {
+
```

Comment:
Extra `\n` here, I see that's your code style in the following functions.

Reply:
Adds a little more whitespace to the functions.

Reply:
Done, thanks.